include/picongpu/plugins/particleCalorimeter/ParticleCalorimeter.hpp

```diff
@@ -285,8 +285,9 @@ class ParticleCalorimeter : public ISimulationPlugin
         this->allGPU_reduce = AllGPU_reduce(new pmacc::algorithm::mpi::Reduce<simDim>(zone_allGPUs));
 
         /* calculate rotated calorimeter frame from posYaw_deg and posPitch_deg */
-        const float_64 posYaw_rad = this->posYaw_deg * float_64(M_PI / 180.0);
-        const float_64 posPitch_rad = this->posPitch_deg * float_64(M_PI / 180.0);
+        constexpr float_64 radsInDegree = pmacc::algorithms::math::Pi<float_64>::value / float_64(180.0);
```

Comment:
maybe call this `radsToDegree` in order to be more explicit?

Reply:
You mean `degreeToRad`? This is how many rads are there in one degree. Or maybe the name is bad and confused you.